packages/ui/src/components/va-alert/useAlertStyles.ts

```diff
@@ -20,6 +20,8 @@ type AlertStyleProps = {
 
 export const useAlertStyles = (props: AlertStyleProps) => {
   const dark = 'var(--va-dark)'
+  const white = '#ffffff'
```

Comment:
I mean, that we have css-variable `--va-white`, so
`const white = 'var(--va-white)'`

Reply:
done